R/constants.R

```diff
@@ -19,6 +19,7 @@ FLAG_PROTECT_FLOAT <- "FLAG_PROTECT_FLOAT"
 # Time constants
 HISTORICAL_YEARS <- 1971:2010
 FUTURE_YEARS <- seq( 2015, 2100, 5 )
+IMF_FUTURE_YEARS <- seq( 2010, 2020, 1 )
```

Comment:
There's an "Add your reaction" pop-up menu available in the upper-right corner of every comment.